pkg/models/job.go

```diff
@@ -43,6 +44,7 @@ const (
 	JobActionRestart     JobAction = "restart"
 	JobActionDelete      JobAction = "delete"
 	JobActionForceDelete JobAction = "force-delete"
+	JobActionRun         JobAction = "run"
```

Comment:
We need to implement validation in the Create method of the build service.

A `run` action job can not be created if the resource type is not job. Same thing for other "invalid" actions.